packages/graphql-api/package.json

```diff
@@ -60,7 +61,6 @@
     "@types/jest": "25.2.3",
     "@types/request-ip": "0.0.35",
     "concurrently": "5.2.0",
-    "graphql": "14.6.0",
```

Comment:
as it is now in peerDependencies we also need to have it as a devDependency otherwise we won't be able to import in in dev mode, can you please revert this line?

Reply:
Ok done. Also added support for v15, like in #1088.